Lib/test/test_exceptions.py

```diff
@@ -674,6 +674,14 @@ def test_str(self):
     def testExceptionCleanupNames(self):
```

Comment:
While you're here, could you rename this to test_exception_cleanup_names  (and any other tests you see with camel case names?)

(Make sure test names are unique after your change.)

Reply:
I think renaming (except for possibly this one test) should happen in a different commit.